docs/deploy.sh

```diff
@@ -98,6 +98,8 @@ redirects[distribution/webhooks]=build-reference/build-webhook/
 redirects[distribution/turtle-cli]=classic/turtle-cli/
 redirects[distribution/app-signing]=app-signing/app-credentials/
 redirects[guides/adhoc-builds]=archived/adhoc-builds/
+# clients is now development
+redirects[clients]=development/
```

Comment:
Will this work for all pages in the `development/` path? Or does each one need to be set specifically, i.e. :

```
redirects[clients/introduction]=development/introduction
redirects[clients/getting-started]=development/getting-started
```

(Also related- "development" seems like a very general path to give, imo it might be better to give it something a bit more specific to)

Reply:
@jonsamp Unfortunately, S3 doesn't let us do wildcards. So for every link that's broken (every file that was there) there should be a separate redirect. It's good that you used server-side redirects though, way better for SEO